pkg/openstack/client/mocks/client_mocks.go

```diff
@@ -0,0 +1,156 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/gardener/gardener-extension-provider-openstack/pkg/openstack/client (interfaces: Factory,Compute)
+
+// Package mocks is a generated GoMock package.
+package mocks
```

Comment:
I'm wondering if we should move all mock packages to `pkg/mock` with subdirectories for the `client` or any other  mock. `g/g` is doing it also this way. https://github.com/gardener/gardener/tree/master/pkg/mock

Reply:
What i don't like about this:
a) if we need changes to the provider and need to use a new openstack service not mentioned here, we would need a g/g release.
b) I do not consider these interfaces as stable to move them into the central repo. In comparison the mcm-provider-openstack uses a slightly different structure. If we consolidate them we could think about it but its premature now

Reply:
My intention was not to move the mocks of the openstack extension to the g/g repo. I wanted to suggest to move all the mocks of the openstack extension to a central place like `pkg/mocks` in the openstack extension repo. Then the structure would be similar like in the g/g repo. :)

Reply:
I misunderstood. This is possible, yes. We can open an issue and give it a try